pkg/cmd/init.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright © 2019 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/DopplerHQ/cli/pkg/configuration"
+	"github.com/DopplerHQ/cli/pkg/http"
+	"github.com/DopplerHQ/cli/pkg/printer"
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+// repoConfigFileName (doppler.yaml)
+const projectTemplateFileName = "doppler.yaml"
+
+var initCommand = &cobra.Command{
+	Use:   "init",
+	Short: "Automatically set up projects in your Doppler dashboard",
```

Comment:
What's the ultimate goal of this command? Is it to support importing a template file, is it to set up a workplace, set up a specific project, etc? I think we can make this description a bit more precise.

Reply:
What about:
 
Scaffold multiple environments and secrets for a repository using a Doppler project template.

Reply:
I like that, but you can also setup multiple projects if you would like. What about:
`Scaffold projects, environments and secrets for a repository using a Doppler project template.`

Reply:
Ah right, I wasn't aware you could set up multiple projects.

What's the use case there?

Reply:
It's more just built in a scalable way that if this would every be needed in the future this is possible (without having to keep backward compatibility etc).

Reply:
1 use case for instance could be a repo with a server & client. Where you want to have them as 2 projects in Doppler.

Reply:
Resolved offline